### PR TITLE
[qa] Add a check for empty line #66

### LINKED ISSUE #66: [qa] Add a check for empty line at the end of a file

### PR DESCRIPTION
**Pull request by unsuitable001**
This PR adds the QA of adding an empty line after every file
GCI Task - https://codein.withgoogle.com/dashboard/task-instances/4670648862375936/

Fixes #66

### CODE REVIEW

File: endline-script.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import os
```
[the remark this answers is not in the document]
Comment (unsuitable001) [reply endline-script.py]: Yes. This script is from that repo

[the remark this answers is not in the document]
Comment (unsuitable001) [reply endline-script.py]: You mean, tell travis to import openwisp-utils and then, run my script from that import. Right?

File: .landscape.yml [hunk 2]
```diff
@@ -1,3 +1,4 @@
 ignore-paths:
     - /docs
-    - /tests
\ No newline at end of file
+    - /tests
+
```
[the remark this answers is not in the document]
Comment (unsuitable001) [reply .landscape.yml]: No. It's not allowing. I'm just checking if the last char is `\n`. Any other thing will raise error. Let me check.

File: endline-script.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (atb00ker) [reply endline-script.py]: No. Check how

[the remark this answers is not in the document]
Comment (unsuitable001) [reply endline-script.py]: Ok

[the remark this answers is not in the document]
Comment (unsuitable001) [reply endline-script.py]: Oh. Now I got it.
One more question. I need to `import subprocess`
Should I do that at the starting of `qa.py`?
If yes, tell me the right sequence to import `subprocess `
(just to overcome one more travis failure.  :persevere:)

[the remark this answers is not in the document]
Comment (atb00ker) [reply endline-script.py]: Run `isort` and `flake8` to avoid errors! :smile:

**Pull request closed without merging**